src/steiner_tree.rs

```diff
@@ -207,7 +310,7 @@ pub fn steiner_tree(
         .iter()
         .map(|x| x.iter())
         .flatten()
-        .copied()
+        .cloned()
```

Comment:
Is there a reason this needs to be changed? I think `copied()` should still work and be more clear on the intent (I don't think for a `usize` there is much of a performance difference for `copy()` vs `clone()`.

Reply:
Yeah, you're right. There is no reason for the change. I tried something that didn't work due to our MSRV requirement and apparently i did not fully reverted in 9754282.